modules/complete-assisted-installer-oci.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc [Using the Assisted Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: PROCEDURE
+[id="complete-assisted-installer-oci_{context}"]
+= Completing the remaining Assisted Installer steps
```

Comment:
We should create and boot instances with ISO before continue installation procedure in AI SaaS 
I think this step should be a chapter since it is important 
(Go to Services > Instances on OCI. Select your compartment, and then click the Create instance button. See [Creating an instance](https://docs.public.oneportal.content.oci.oraclecloud.com/en-us/iaas/Content/Compute/Tasks/launchinginstance.htm#top) in the Oracle documentation.)
Because there are no nodes in Host discovery until instances are up and agent are identify network configuration.

Reply:
> @yobshans . Any additional steps needed to explicitly boot the instance?

there should be nothing to do, the instances should be booted after the stack is created